### PR TITLE
terraform file cleanup #785

### LINKED ISSUE #785: terraform file cleanup
**Describe the solution you'd like**
In `dlp-terraform/dlp.tf`, we currently dump our S3 + dynamodb table definitions in this file. This creates a bit of confusion in where to identify configurations of these AWS resources. For better organization, we should try to move the dynamodb table definitions into its own `.tf` file and the S3 definitions from `dlp.tf` into its own terraform file. 

**Additional context**
@thomaschin35 to confirm if this is a permitted design for terraform

**Setup Instructions (what branch to work off of)**
Run the following commands
```
git checkout nextjs
git pull origin nextjs
git checkout -b dlp-tf-cleanup
```
FYI: If you are not able to immediately run `git checkout nextjs`, make sure you commit your changes in the current branch or run `git stash` and then execute the above commands

### PR DESCRIPTION
**Pull request by AmyCGT**

### CODE REVIEW

File: dlp-terraform/S3.tf [hunk 1]
```diff
@@ -0,0 +1,77 @@
+terraform {
```
Comment (karkir0003) [inline dlp-terraform/S3.tf]: make `S3` lowercase.

`S3.tf` --> `s3.tf`

**Review by karkir0003 — COMMENTED** [review]
looks good to me. Let's have @thomaschin35 take a look

**Comment by farisdurrani** [comment]
lgtm, but I'll leave Thomas or Karthik to approve

**Comment by karkir0003** [comment]
> lgtm, but I'll leave Thomas or Karthik to approve

so Amy needs to fix up a minor change, but I think Thomas should be sole approver on this PR since he's the most knowledgeable on Terraform @farisdurrani

**Comment by thomaschin35** [comment]
@AmyCGT it all looks good to me, make sure to check if they're correct by doing `terraform validate` in each of the corresponding folders. Once that is all good, this pr is good to go.

**Review by thomaschin35 — APPROVED** [review]

**Comment by karkir0003** [comment]
@AmyCGT give this a try

**Comment by karkir0003** [comment]
@thomaschin35 can we setup a github action to autovalidate terraform schemes "like run terraform validate" within each individual tf file?

**Comment by AmyCGT** [comment]
@thomaschin35 @karkir0003 it's telling me to run terraform init for some required provider thing? would i have to do that in each folder?

**Comment by thomaschin35** [comment]
@AmyCGT yes whenever you start up your terminal you have to run terraform init to initialize the environment. So just run `terraform init`, and then `terraform validate `

**Comment by AmyCGT** [comment]
@thomaschin35 ok. i validated them both, there seems to be changes made in files? should i commit those or leave them

**Comment by karkir0003** [comment]
you cd into dlp-terraform and run terraform init there itself i believe @AmyCGT

**Comment by karkir0003** [comment]
> @thomaschin35 ok. i validated them both, there seems to be changes made in files? should i commit those or leave them

dont deploy any changes to aws. this task is just a cleanup

**Comment by karkir0003** [comment]
@thomaschin35 can you confirm?

**Comment by thomaschin35** [comment]
Yes don't commit anything, those files are just from running terraform init (start up files), and yea don't deploy anything, so don't run `terraform apply`

**Comment by AmyCGT** [comment]
wasnt able to run `terraform init` in dlp-terraform so had to do it in dynamodb and s3 folders

**Comment by AmyCGT** [comment]
s3 validation:
image
dynamodb validation:
image

**Comment by karkir0003** [comment]
@AmyCGT awesome. Since thomas approved, you can merge!

**Pull request merged**